kie-server-parent/kie-server-services/kie-server-services-case-mgmt/src/main/java/org/kie/server/services/casemgmt/CaseManagementServiceBase.java

```diff
@@ -160,6 +160,22 @@ public void cancelCaseInstance(String containerId, String caseId, boolean destro
         }
     }
 
+    public void reopenCase(String caseId, String containerId, String caseDefinitionId, String payload, String marshallingType) {
+        containerId = context.getContainer(containerId, LatestContainerLocator.get()).getContainerId();
```

Comment:
I was thinking about this and though it would make sense to always start process from latest deployment but after giving it a second thought (and your comment) I think you're right as it would be safer to stay in same container as the case itself. Will update it